docs/tutorials/tutorial-rollup.md

```diff
@@ -49,150 +49,101 @@ For this tutorial, we'll use a small sample of network flow event data, represen
 {"timestamp":"2018-01-02T21:35:45Z","srcIP":"7.7.7.7", "dstIP":"8.8.8.8","packets":12,"bytes":2818}
 ```
 
-A file containing this sample input data is located at `quickstart/tutorial/rollup-data.json`.
+The tutorial guides you through how to ingest this data using rollup.
```

Comment:
This line is repetitive. Accept Victoria's suggestion and move this line to the introduction. Replace line 26 (This tutorial demonstrates the effects of rollup on an example dataset.) with this line.